BUILD.gn

```diff
@@ -675,6 +676,87 @@ source_set("electron_lib") {
   if (enable_electron_extensions) {
     sources += filenames.lib_sources_extensions
   }
+
+  if (enable_pdf_viewer) {
+    deps += [
+      "//components/pdf/browser",
+      "//components/pdf/renderer",
+      "//pdf",
+    ]
+  }
+}
+
+source_set("plugins") {
```

Comment:
Can you move this to `chromium_src/BUILD.gn`, better to keep all `//chrome` references in one place. Maybe it could now be `chromium_src` => `chrome_src`